docs/tutorials/hello-world-example.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Hello World Example
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This tutorial will demonstrate how to get started with Rancher Desktop by pushing an app to a local Kubernetes cluster.
+
+Rancher Desktop works with two container engines, [`containerd`](https://containerd.io/) and [Moby](https://mobyproject.org/), the open-sourced components of the Docker ecosystem. For `nerdctl`, use the containerd runtime. For `docker`, use the Moby (`dockerd`) runtime.
+
+### Hello World example
+
+**Create a folder as follows:** Make sure to specify your "USERNAME". For this example, USERNAME = John.
+```
+mkdir ../hello-world
+cd ../hello-world
+```
+
+**Create a Dockerfile with the following content:**
+```
+FROM alpine  
+CMD ["echo", "Hello, John!"]
+```
+
+**Build and run the image for verification purposes:**
+
+<Tabs groupId="container-runtime">
+  <TabItem value="nerdctl" default>
+
+```
+nerdctl build -t John/hello:1.0 .
```

Comment:
**J**ohn/hello needs to be fully lower case and all other commands need to be changed accordingly

Reply:
Updated as suggested.